app/assets/stylesheets/unimported/email.css

```diff
@@ -0,0 +1,91 @@
+body {
+  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  color: #3f3f3f;
+  background-color: #f1f1f1;
+  padding: 10px 24px;
+}
+
+table {
+  min-width: 400px;
+  border: 1px solid #cccccc;
+  padding: 5px;
+  background-color: #ffffff;
+  border-collapse: collapse;
+  text-align: justify;
+  width: 100%;
+}
+
+table.center {
+  text-align: center;
+}
+
+th {
+  text-align: left;
+  border-collapse: collapse;
+  background: #f5f5f5;
+  background-image: linear-gradient(#f5f5f5, #e8e8e8);
+  padding: 4px;
+  border: 1px solid #cccccc;
+}
+
+td.hosts-rows {
+  text-align: left;
+  border-collapse: collapse;
+  background-color: #FFFFFF;
+  padding: 4px;
+  border: 1px solid #cccccc;
+}
+
+td.dashboard {
+  font-size: 125%;
+  width: 33%;
+  padding: 5px;
+  border: 1px solid #ccc;
+}
+
+div.dashboard {
+  background: #e1e2e3;
+  padding: 20px 40px;
+  margin: 5px 0px 10px;
+}
+
+p.dashboard {
```

Comment:
what about `td.dashboard p` ?   p element is a child of td element with `dashboard` class.